setup.cfg

```diff
@@ -1,4 +1,4 @@
 [nosetests]
 exclude=(django)
-with-coverage=1
-cover-package=couchdbkit
+# with-coverage=1
+# cover-package=couchdbkit
```

Comment:
Is this just disabling coverage tracking? Any particular reason?

Reply:
Think that was just a local thing. I'll add it back before taking off open for review